Directory.Packages.props

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <ItemGroup>
```

Comment:
Which of these packages aren't using the latest versions?

Reply:
Latest versions, and if there was a newer major version included both latest of the current major version + latest major version.

| Package | Current Version | Latest Version |
| --------- | ----------------- | --------------- |
|Microsoft.Azure.WebJobs.Extensions.Storage | 4.0.3 | 4.0.5/5.0.0 |
|Microsoft.NET.Sdk.Functions | 3.0.13 | latest/4.1.0 |
|Microsoft.ApplicationInsights | 2.17.0 | 2.20.0 |
|Microsoft.Azure.WebJobs | 3.0.31 | latest |
|Microsoft.Data.SqlClient | 3.0.1 | latest/4.1.0 |
|Microsoft.SourceLink.GitHub | 1.1.1 | latest |
|morelinq | 3.3.2 | latest |
|System.Runtime.Caching | 4.7.0 | latest/6.0.0 |
|Microsoft.NET.Test.Sdk | 17.0.0 | 17.1.0 |
|Moq | 4.14.3 | 4.17.1 |
|xunit | 2.4.0 | 2.4.1 |
|xunit.runner.visualstudio | 2.4.0 | 2.4.3 |